config/environments/production.rb

```diff
@@ -76,4 +76,5 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+  config.jbuilder.prettify = true
```

Comment:
Can't we move this into `application.rb` to avoid duplication here and in development.rb?